src/Compilers/CSharp/Test/Symbol/Symbols/CustomModifiersTests.cs

```diff
@@ -57,27 +57,21 @@ static void Main()
             var compilation = CreateCompilationWithCustomILSource(source, ilSource, options: TestOptions.ReleaseExe);
 
             var test = compilation.GetTypeByMetadataName("Test1").GetMember<MethodSymbol>("Test");
-            var type = (INamedTypeSymbol)test.Parameters.First().Type;
+            var type = (NamedTypeSymbol)test.Parameters.First().Type.TypeSymbol;
             Assert.Equal("System.Int32 modopt(System.Runtime.CompilerServices.IsLong)?", type.ToTestDisplayString());
-            Assert.Equal("System.Runtime.CompilerServices.IsLong", type.GetTypeArgumentCustomModifiers(0).Single().Modifier.ToTestDisplayString());
-            Assert.Throws<System.IndexOutOfRangeException>(() => type.GetTypeArgumentCustomModifiers(1));
```

Comment:
The equivalent assert would be `Assert.Throws<IndexOutOfRangeException>(() => type.TypeArguments[1].CustomModifiers)`. It seemed unnecessary to test `TypeArguments[1]` for `Nullable<int>`.